WinCCOA_QualityChecks/msg/en_US.utf8/QgBase.cat

```diff
@@ -1,6 +1,7 @@
 00001,QualityGates does not run successfully !!!
 00010,Assertion return error: $1
 00011,Assertion works: $1
+00012,Assertion return an acceptable error (known bug): $1
```

Comment:
```suggestion
00012,Assertion returns an acceptable error (known bug): $1
```

Reply:
thx for review, but you can keep it now. Is only draft PR. Many other changes comming soon. Because there are still too much waste scripts.